test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/InstallCommandTests.cs

```diff
@@ -258,52 +255,5 @@ public static CommandRunnerResult RunInstall(string nugetExe, SimpleTestPathCont
 
             return r;
         }
-
-        [Fact]
-        public async Task Install_WithHttpSource_Warns()
```

Comment:
I moved this test into a different class, because if you look at the other tests in this class they are all signing related *CI* only tests.